src/archanaNJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,23 @@
+package archanaNJun20;
+
+public class PalindromeNumber {
+
+	void printPalindromeNumber(int number){
+		int oNumber=number;
+		int rev=0;
+		while(number>0){
+			int mod=number%10;
```

Comment:
Never initialize variable in loop. and also no need of this variable int mod. 
revNumber = (revNumber*10) + (number%10);
number=number/10;